src/components/events/Events.js

```diff
@@ -200,30 +228,89 @@ const Events = () => {
           onButtonClick={loadGetEvents}
           onResetSearch={handleRefreshSearch}
         />
+        <br />
+        <Typography
+          style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+        >
+          Filters
+        </Typography>
+        <DropdownFilter
+          placeholder="Select the Status"
```

Comment:
I think setting the default value of the status filter to be "Active" would be helpful. I think majority of the time, users would want to see the active upcoming events

Reply:
I disagree, the user should be able to view all the events and then filter as needed

Reply:
i am fine with either - i don't think implementation is too hard but then again, we need to remember that these are filters.

if we want a default look, its better if we render the events like that in the first place IMO

Reply:
I still disagree, but I'm outvoted here so its fine

Reply:
> I disagree with your disagreement. Who wants to see a cancelled event as a default? Why would you want to look a list of events, see one you are really interested in, get excited for it, see participants aren't full, and then notice its cancelled. Why would anyone want to see an event that is no longer happening??? I get that there may be use cases, but I definitely don't go search up and look at a list of events wanting to look at ones that aren't even happening.

it depends on what we want to show the user on this page. Is this page a list of events both previous and upcoming? Or is it only just a list of events that are upcoming? Ideally, the user is able to make this choice through the filters, and that's why the filter is placed above the list, so that the user makes this decision and then looks at the list. If the user is looking at all the events, we clearly indicate on the card for the event if it's cancelled or not. 

Either way, as long as both cancelled events and active events can show up on this page, I have no issues with showing active events on first render. @gabbytang I'll let you decide since you implemented the events feature :disguised_face:

Reply:
> > I disagree with your disagreement. Who wants to see a cancelled event as a default? Why would you want to look a list of events, see one you are really interested in, get excited for it, see participants aren't full, and then notice its cancelled. Why would anyone want to see an event that is no longer happening??? I get that there may be use cases, but I definitely don't go search up and look at a list of events wanting to look at ones that aren't even happening.
> 
> it depends on what we want to show the user on this page. Is this page a list of events both previous and upcoming? Or is it only just a list of events that are upcoming? Ideally, the user is able to make this choice through the filters, and that's why the filter is placed above the list, so that the user makes this decision and then looks at the list. If the user is looking at all the events, we clearly indicate on the card for the event if it's cancelled or not.
> 
> Either way, as long as both cancelled events and active events can show up on this page, I have no issues with showing active events on first render. @gabbytang I'll let you decide since you implemented the events feature 🥸

This ain't my pr, I have no hold here other than my opinion. I can see where you both are coming from, though I vehemently disagree with that being the viewpoint in which an end user will hold perspective when using the platform. I 
 personally would expect a "social media" platform to show me upcoming events that I would like to interact and join, having cancelled or passed events be one of the first things I see would not be realistically helpful to me. I would in fact enjoy the option to see cancelled events, but not on initial rendering.